i_scene_cp77_gltf/__init__.py

```diff
@@ -1,7 +1,7 @@
 bl_info = {
     "name": "Cyberpunk 2077 IO Suite",
-    "author": "HitmanHimself, Turk, Jato, dragonzkiller, kwekmaster, glitchered, Simarilius, The Magnificent Doctor Presto",
-    "version": (1,4, 0),
+    "author": "HitmanHimself, Turk, Jato, dragonzkiller, kwekmaster, glitchered, Simarilius, Doctor Presto",
```

Comment:
shotc was in here wasnt he?

Reply:
missed that! I merged his changes manually but didn't think to check that one. Fixed it